rosdep/python.yaml

```diff
@@ -582,12 +571,7 @@ python-empy:
     xenial: [python-empy]
 python-enum:
   fedora: [python-enum]
-  ubuntu:
-    precise: [python-enum]
-    quantal: [python-enum]
-    raring: [python-enum]
-    saucy: [python-enum]
-    trusty: [python-enum]
+  ubuntu: [python-enum]
```

Comment:
Possible that on Xenial, this should resolve to python-enum34, given that the two packages are now in conflict.